.github/workflows/vulnerability-scanner-tests.yml

```diff
@@ -29,87 +29,94 @@ jobs:
       - name: Calculate deltas
         run: |
 
-          DIFFS=$(git diff --name-only origin/${{ github.head_ref }} origin/${{ github.base_ref }})
+          DIFFS=""
+          
+          # Get the list of modified files
+          if [ -z ${{ env.ACT }} ]; then
+            # CI execution
+            DIFFS=$(git diff --name-only HEAD origin/${{ github.base_ref }})
+            # Local execution
+          else
+            if [ -n "${{ env.REF_BRANCH }}" ]; then
+              # If REF_BRANCH is set, use it to calculate the diff
```

Comment:
Question: Where is REF_BRANCH set?

Reply:
I've updated the PR description. Is set via command line when running act:

`act -W <path_to_workflow> --artifact-server-path <artifacts_location> --env REF_BRANCH=<base_branch (optional)>`